### ISSUE TITLE
fail to load pytorch model

### ISSUE THREAD

**Issue by littletree123** [triton-inference-server/server#2672]
hi, i try to load an age-estimation model but it failed. the information is not enough so that i can't find my wrong.
my env pytorch 1.6.0. and i failed to load the model. i think maybe is because i use the function 'save' rather than 'jit.save' my pytorch verison mismatched and i transfer my torch model to onnx, but it still failed to load. 
and **save model code is** 

> 

            model_state_dict = model.module.state_dict() if args.multi_gpu else model.state_dict()
            torch.save(
                {
                    'epoch': epoch + 1,
                    'arch': cfg.MODEL.ARCH,
                    'state_dict': model_state_dict,
                    'optimizer_state_dict': optimizer.state_dict()
                },
                str(checkpoint_dir.joinpath("epoch{:03d}_{:.5f}_{:.4f}.pth".format(epoch, val_loss, val_mae)))
            )

**transfer code is** 

> import torch
from model import get_model
from defaults import _C as cfg
device = torch.device("cuda" if torch.cuda.is_available() else "cpu")

model = get_model(model_name=cfg.MODEL.ARCH, pretrained=None)
checkpoint = torch.load("checkpoint/epoch061_0.02474_4.6416.pth", map_location="cpu") # pytorch模型加载
model.load_state_dict(checkpoint['state_dict'])
model=model.to(device)
batch_size = 2  #批处理大小
input_shape = (3,7,7)   #输入数据,改成自己的输入shape

# #set the model to inference mode
model.eval()

x = torch.randn(batch_size, *input_shape)   # 生成张量
x = x.to(device)
export_onnx_file = "test.onnx"  # 目的ONNX文件名
torch.onnx.export(model,
                    x,
                    export_onnx_file,
                    opset_version=10,
                    do_constant_folding=True, # 是否执行常量折叠优化
                    input_names=["input"], # 输入名
                    output_names=["output"], # 输出名
                    dynamic_axes={"input":{0:"batch_size"},  # 批处理变量
                                    "output":{0:"batch_size"}})

**my config.pbtxt is** 

> name: "resnet50"
platform: "onnxruntime_onnx"
max_batch_size: 1
input [
  {
    name: "layer0.conv1"
    data_type: TYPE_FP32
    format: FORMAT_NCHW
    dims: [ 1, 3, 7, 7]
  }
]
output [
  {
    name: "last_linear"
    data_type: TYPE_FP32
    dims: [ 1, 101, 2048 ]
  }
]
instance_group {
  name: "resnet50"
  count: 1
  gpus: 0
  kind: KIND_GPU
}

and the command is
docker run --rm --shm-size=1g -p8000:8000 -p8001:8001 -p8002:8002 -v/home/cjl/hust/tensorRT/server/docs/examples/model_repository:/models nvcr.io/nvidia/tritonserver:20.09-py3 tritonserver --model-repository=/models --model-control-mode=explicit --load-model resnet50

and the print is:
=============================
== Triton Inference Server ==
=============================

NVIDIA Release 20.09 (build 16016295)

Copyright (c) 2018-2020, NVIDIA CORPORATION.  All rights reserved.

Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.
NVIDIA modifications are covered by the license terms that apply to the underlying
project or file.

I0328 02:46:41.270994 1 metrics.cc:184] found 1 GPUs supporting NVML metrics
I0328 02:46:41.276620 1 metrics.cc:193]   GPU 0: GeForce GTX 1080 Ti
I0328 02:46:41.277820 1 server.cc:120] Initializing Triton Inference Server
I0328 02:46:41.277849 1 server.cc:121]   id: 'triton'
I0328 02:46:41.277858 1 server.cc:122]   version: '2.3.0'
I0328 02:46:41.277869 1 server.cc:128]   extensions:  classification sequence model_repository schedule_policy model_configuration system_shared_memory cuda_shared_memory binary_tensor_data statistics
I0328 02:46:41.594145 1 pinned_memory_manager.cc:195] Pinned memory pool is created at '0x7f2706000000' with size 268435456
I0328 02:46:41.594680 1 cuda_memory_manager.cc:98] CUDA memory pool is created on device 0 with size 67108864
E0328 02:46:41.601811 1 model_repository_manager.cc:1604] model input NHWC/NCHW require 3 dims for resnet50
I0328 02:46:41.601870 1 server.cc:213] Waiting for in-flight requests to complete.
I0328 02:46:41.601883 1 server.cc:228] Timeout 30: Found 0 live models and 0 in-flight non-inference requests
error: creating server: Internal - failed to load all models

i don't know how to debug. thanks so much!!

**Comment by littletree123** [comment]
i changed the config.pbtxt and it run successfully! but when i run
./image_client -m resnet50 /pic.jpg
print the information:
**expecting model output to be a vector**

**Comment by niqbal996** [comment]
HI @littletree123, 
Did you manage to solve this error? 
I am also expecting a mask image as the output of the inference request from the server in [3, 512, 512] dimension but the image_client.py is giving me the same error, 
**expecting model output to be a vector**
